optima/model.py

```diff
@@ -105,6 +107,8 @@ def model(simpars=None, settings=None, verbose=2, safetymargin=0.8, benchmark=Fa
     circum   = simpars['circum']    # Prevalence of circumcision (P)
     stiprev  = simpars['stiprev']   # Prevalence of STIs (P)
     prep     = simpars['prep']      # Prevalence of PrEP (P)
+    numpmtct = simpars['numpmtct']  # Number (or proportion?) of people receiving PMTCT (P/N)
+    usepmtctprop=True if all(numpmtct<1) else False
```

Comment:
it still feels very fragile and hacky to me; i'd prefer to have separate parameters, but that has its pitfalls too. i don't have a better suggestion so i'm ok to leave it as is for now.